neuraxle/pipeline.py

```diff
@@ -55,13 +60,73 @@ def inverse_transform(self, processed_outputs):
         return self.inverse_transform_processed_outputs(processed_outputs)
 
 
+class PipelineRepository(ABC):
+    def __init__(self, cache_folder, pipeline_cache_list_file_name='pipeline_cache_list.txt'):
+        self.pipeline_cache_list_file_name = pipeline_cache_list_file_name
+        self.cache_folder = cache_folder
+
+    def save(self, pipeline: 'Pipeline', data_container: DataContainer):
+        pipeline_cache_folder = os.path.join(self.cache_folder, pipeline.name)
+
+        if not os.path.exists(pipeline_cache_folder):
+            os.makedirs(pipeline_cache_folder)
+
+        with open(os.path.join(pipeline_cache_folder, self.pipeline_cache_list_file_name), mode='a') as file:
```

Comment:
The `file` word is already used for something in python. True = False. I suggest `f` or `fp` or `_file` or `file_`.